examples/memeolist.tables.sql

```diff
@@ -12,9 +12,8 @@ CREATE TABLE profile (
 CREATE TABLE meme (
   id SERIAL NOT NULL PRIMARY KEY,
   photourl CHARACTER VARYING(500) NOT NULL,
-  owner CHARACTER VARYING(100) NOT NULL,
   likes NUMERIC NOT NULL,
-  ownerid SERIAL NOT NULL references profile(id)
```

Comment:
Really happy with this PR except this one thing. I strongly believe this field should still be `ownerid` or `ownerId`. (whereas at a higher level, the resolved object will be called `owner`)

Reply:
Resolvers are now overly simplified (`select * from meme`) so table and graphql names are connected.

Reply:
Okay I will concede that one 😄